bff-nav_ansatt/src/main/kotlin/no/nav/amt/tiltak/bff/nav_ansatt/dto/Status.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.amt.tiltak.bff.nav_ansatt.dto
+
+import no.nav.amt.tiltak.core.domain.tiltak.Gjennomforing
+
+enum class Status {
+	IKKE_STARTET, GJENNOMFORES, AVSLUTTET
```

Comment:
Er det en fordel med egen statuser enum som er lik gjennomføring statusene? Virker rendundant med ekstra steg når man vil gjøre endringer. Hvis man ønsker begge så bør denne kanskje hete StatusDto

Reply:
Jeg har bare flyttet den som var der fra før ut i en egen fil siden den skulle brukes flere steder nå :) Jeg antar at grunnen til at dere har egne objekter som brukes i respons fra apiet er for at man skal kunne gjøre endringer i strukturen videre bakover i appen uten at det påvirker apiresponsen. Men jeg er enig i at det er lite sannsynlig at man vil ønske å f.eks. legge til nye verdier i enumet som brukes bakover i appen uten at den nye verdien også skal vises i apiet,  så jeg kan sikkert endre sånn at vi bare har et status-enum 😊